easybib/providers/nginx.rb

```diff
@@ -41,6 +41,11 @@
 
   ::Chef::Resource.send(:include, PhpFpm::Helper)
 
+  execute 'nginx_configtest' do
```

Comment:
Ah, and "When is this executed in a loop?" — E.g. in stacks where multiple apps share one server. For initial setups.